src/clean_env_before_mining.py

```diff
@@ -10,6 +10,7 @@
 import os
 import shutil
 
+DATA_DIRECTORY = "../data"
```

Comment:
The controller file should define the data path by adding a predefined value in the env_vars dict.

Reply:
Same, will come to this later.